NEXTCLOUD.cmake

```diff
@@ -15,6 +15,34 @@ set( APPLICATION_VIRTUALFILE_SUFFIX "nextcloud" CACHE STRING "Virtual file suffi
 set( LINUX_PACKAGE_SHORTNAME "nextcloud" )
 set( LINUX_APPLICATION_ID "${APPLICATION_REV_DOMAIN}.${LINUX_PACKAGE_SHORTNAME}")
 
+option(DEV_BUILD "If enabled, name variables will be appended with DEV_SUFFIX in order to reduce conflicts." OFF)
+if (NOT DEV_SUFFIX)
+  set(DEV_SUFFIX "Dev"
+    CACHE STRING
+      "Choose a suffix with which to append build name variables to prevent conflicts."
+  FORCE)
```

Comment:
Please format the code in the standard CMake format, like:
```cmake
set(DEV_SUFFIX 
      "Dev"
      CACHE STRING
      "Choose a suffix with which to append build name variables to prevent conflicts."
      FORCE)
```

Reply:
I literally copied and pasted this from DefineCMakeDefaults.cmake:
https://github.com/nextcloud/desktop/blob/6cdae6c19edca2a0847f55f44f5290ee4a937d5b/cmake/modules/DefineCMakeDefaults.cmake#L28-L34

But, yes, I can standardize the indentation.